### PR TITLE
Enabled paused gamespeed

### PR DESCRIPTION
**Pull request by DJ1TJOO**
### Enabled paused gamespeed
This PR enables the paused gamespeed so you can build, but all processes are paused. It's bound to F7.

### CODE REVIEW

File: src/js/game/time/game_time.js [hunk 1]
```diff
@@ -185,6 +190,19 @@ export class GameTime extends BasicSerializableObject {
         this.speed = speed;
     }
 
+    togglePause() {
+        if (this.getIsPaused()) {
```
Comment (tobspr) [inline src/js/game/time/game_time.js:194]: I wouldn't do that many checks here, just do `this.setSpeed(new RegularGameSpeed())`

**Comment by tobspr** [comment]
This would be nice as a mod

File: src/js/game/time/game_time.js (hunk 1 above)
Comment (DJ1TJOO) [reply src/js/game/time/game_time.js:194]: I made it like this so there can be other gamespeeds then just regular speed

**Comment by DJ1TJOO** [comment]
Adding it as a mod when modloader is done

**Pull request closed without merging**